models/resource.js

```diff
@@ -9,11 +9,11 @@ module.exports = (sequelize, DataTypes) => {
      */
     static associate(models) {
       // define association here
+      Resource.belongsTo(models.Category);
     }
   }
   Resource.init(
     {
-      CategoryId: DataTypes.INTEGER,
```

Comment:
Because of the "belongsTo" definition above, we no longer need to specify CategoryId as a separate attribute of the Resource model.

Instead, the belongsTo relationship tells Sequelize to _expect_ a CategoryId column for that association. This is why it was important for the name of the column to be _exactly_ CategoryId.